src/main/java/model/Monster.java

```diff
@@ -71,6 +71,7 @@ public void checkCollision(final Bubble bubble) {
             double bubbleY = bubble.getY();
             double bubbleX2 = bubbleX + getWidth();
             double bubbleY2 = bubbleY + getHeight();
+            System.out.println();
```

Comment:
Please check that no println statements or comments used for debugging are still left in the code.